hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ReconTaskControllerImpl.java

```diff
@@ -158,18 +158,20 @@ private void ignoreFailedTasks(List<ReconOmTask.TaskResult> failedTasks) {
   }
 
   @Override
-  public synchronized void reInitializeTasks(ReconOMMetadataManager omMetadataManager) {
+  public synchronized void reInitializeTasks(ReconOMMetadataManager omMetadataManager,
+                                             Map<String, ReconOmTask> reconOmTaskMap) {
     Collection<NamedCallableTask<ReconOmTask.TaskResult>> tasks = new ArrayList<>();
+    Map<String, ReconOmTask> localReconOmTaskMap = reconOmTaskMap;
+    if (null == reconOmTaskMap) {
```

Comment:
nit
```suggestion
    if (reconOmTaskMap == null) {
```

Reply:
Thanks @dombizita for your review. I have addressed the comments. Please have a relook.